content/en/serverless/azure_app_services/azure_app_services_windows.md

```diff
@@ -328,6 +303,178 @@ Learn more about [custom metrics][10].
 {{% /tab %}}
 {{< /tabs >}}
 
+## Logging
+
+{{< tabs >}}
+{{% tab ".NET" %}}
+
+### Application Logging
+
+You can send logs from your application in Azure App Service to Datadog in one of the following ways:
+1. Use the [installation steps](#installation) above to enable APM with the Datadog APM extension, then [enable Agentless logging][7]
+2. [Agentless logging with the Serilog sink][8]
+
+Both methods allow trace ID injection, making it possible to connect logs and traces in Datadog. To enable trace ID injection with the extension, add the application setting `DD_LOGS_INJECTION:true`.
+
+**Environment Variables for Logging**
+
+Configure these environment variables in your Azure App Service Application Settings for optimal log collection:
+
+| Variable | Description | Example |
+|----------|-------------|---------|
+| `DD_SOURCE` | Identifies the log source for proper processing | `csharp` |
+| `DD_SERVICE` | Your application's service name | `my-web-app` |
+| `DD_ENV` | Your application's environment | `production`, `staging`, `development` |
```

Comment:
`DD_SERVICE` and `DD_ENV` are not logging-specific. I think we should leave them in the "installation" section, maybe better formatting up there, like this table, would be nice.

Reply:
I've since completly DRY'd this all up and moved into the installation section for WIndows stuff

<img width="734" height="967" alt="image" src="https://github.com/user-attachments/assets/215a4bad-050e-4cf1-bc6f-7651a4986994" />